ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -253,3 +253,33 @@ ${local.skip_on_weekends}
     action_group = var.action_group_ids
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "bulk_results_upload" {
+  name                = "${var.env}-bulk_results_upload"
+  description         = "${local.env_title} alert when bulk uploads fail"
+  location            = data.azurerm_resource_group.app.location
+  resource_group_name = var.rg_name
+
+  action {
+    action_group = var.action_group_ids
+  }
+
+  data_source_id = var.app_insights_id
+  enabled        = contains(var.disabled_alerts, "bulk_results_upload") ? false : true
+
+  query = <<-QUERY
+requests
+| where true
```

Comment:
is this doing something?

Reply:
I DON'T KNOW!  But I see this in a lot of our queries so I added it in mine.

Reply:
😂  I assumed it was when `skip_on_weekends` gets evaluated in a query

Reply:
I am fairly certain that it's not doing anything.